tests/scripts/all.sh

```diff
@@ -1197,14 +1196,10 @@ run_all_components () {
     run_component component_test_aes_fewer_tables
     run_component component_test_aes_rom_tables
     run_component component_test_aes_fewer_tables_and_rom_tables
-    if uname -a | grep -F Linux >/dev/null; then
-        run_component component_test_make_shared
-    fi
-    if uname -a | grep -F x86_64 >/dev/null; then
-        run_component component_test_m32_o0
-        run_component component_test_m32_o1
-        run_component component_test_mx32
-    fi
+    run_component component_test_make_shared
```

Comment:
I think `make SHARED=1` is now supposed to work on OSX so `component_test_make_shared` should be unconditional.